packages/server/src/api.ts

```diff
@@ -64,7 +64,13 @@ export class Api {
       cors({
         origin: '*',
         methods: ['GET', 'POST', 'DELETE'],
-        allowedHeaders: ['Content-Type', 'Authorization', 'Content-Length']
+        allowedHeaders: [
+          'Content-Type',
+          'Authorization',
+          'Content-Length',
+          'x-bp-messaging-client-id',
+          'x-bp-messaging-client-token'
```

Comment:
Had to add those headers there otherwise, I would have cors issues when running the tests locally.

Reply:
Can we just allow any header? Why would we need this setting anyway?

Reply:
Can we just allow any header? Why would we need this setting anyway?

Reply:
I think it's good practice to have those there. See that for reference: https://stackoverflow.com/questions/17992042/whats-the-point-of-the-access-control-allow-headers-header

But in our case, since no one will really make requests to messaging from the front-end, we may consider removing them.

Reply:
Yup I'd just remove them if they're not required